src/vs/vscode.proposed.d.ts

```diff
@@ -1005,11 +1005,10 @@ declare module 'vscode' {
 		Code = 2
 	}
 
-	export enum NotebookCellRunState {
-		Running = 1,
-		Idle = 2,
-		Success = 3,
-		Error = 4
+	export interface NotebookCellPreviousRunResult {
+		success?: boolean;
+		duration?: number;
+		executionOrder?: number;
```

Comment:
Should `start()` already/also accept the execution order? That would match the `execute_input` message: https://jupyter-protocol.readthedocs.io/en/latest/messaging.html#code-inputs

Reply:
Yes